docs/en/operation/Scalability-Tuning.md

```diff
@@ -79,16 +110,16 @@ clients and then convert to a operations/second metric
 
 For example, in a deployment with 2 users, 50 Presto worker nodes (with 200 task concurrency), and
 50 Alluxio nodes, the estimations would come out to the following
-* 50 (workers) + 2 (users) x 11 (block pool size) x 10 (file pool size) x 50 (services) = 11050
+* 50 (workers) + 2 (users) x (11 (block pool size) + 10 (file pool size)) x 50 (services) = 2150
```

Comment:
This calculation was wrong. But not sure if I'm now right.

Reply:
I believe the entire `Number of Concurrent Clients` section is a stretch.
I don't think we can provide meaningful numbers here that covers all possible client applications. (Concurrency setting could be executed at a single FS or multiple FSes depending on the application.)

I vote for dropping the section. @yuzhu WDYT?

Reply:
+1 

I don't think this number is ever validated. Now that we have better visibility (and we can add more), we should perform some validation and add a section later which we guarantee the correctness.

Reply:
yeah. i agree